### ISSUE TITLE
org wide mapping

### ISSUE THREAD

**Issue by sils** [johnkferguson/octomaps#34]
would be neat if we could map all contributors of the whole organization, not only one repo

**Comment by johnkferguson** [comment]
I agree, I think it would be a cool feature, but I haven't been actively involved with this project for awhile. I'm open to any PRs if anyone wants to tackle this.
